src/Build/Logging/LoggerDescription.cs

```diff
@@ -10,7 +10,9 @@
 using Microsoft.Build.BackEnd;
 
 using InternalLoggerException = Microsoft.Build.Exceptions.InternalLoggerException;
+#if NETCOREAPP
 using System.Linq;
+#endif
```

Comment:
We can fix this to just actually remove the need: remove `FEATURE_TYPE_GETINTERFACE` since it's available in all targets now.